salt/cni/init.sls

```diff
@@ -48,3 +48,66 @@ include:
       - file:      /etc/kubernetes/addons/kube-flannel-rbac.yaml
 
 {% endif %}
+
+{% set plugin = salt['pillar.get']('cni:plugin', 'cilium').lower() %}
+{% if plugin == "cilium" %}
```

Comment:
There's no need to fetch the pillar again, this can be reduced to:

```SaltStack
{% set plugin = salt['pillar.get']('cni:plugin', 'flannel').lower() %}
{% if plugin == "flannel" %}
# flannel things
{% elsif plugin == "cilium" %}
# cilium things (this whole block)
{% endif %}

Reply:
Thanks for reviewing, Updated.